syntax = "proto3";

package notification;

service NotificationService {
  rpc SendNotification (NotificationRequest) returns (NotificationResponse);
}

message NotificationRequest {
  int32 user_id = 1;
  string message = 2;
}

message NotificationResponse {
  string message = 1;
}
